test/unittests/Loris_PHPUnit_Database_TestCase.php

```diff
@@ -89,6 +89,7 @@ protected function setUp()
      *
      * @return \PHPUnit_Extensions_Database_DB_DefaultDatabaseConnection
      */
+    /* @phan-suppress-next-line PhanUndeclaredTypeReturnType*/
```

Comment:
Why isn't it declared as PHPUnit_Extensions_Database_DB_DefaultDatabaseConnection?

Reply:
I remember that phan just didn't seem to know about it... will re-test and try again.

Reply:
if phan doesn't know about it, that means the PHP doesn't either and phan is flagging a bug